src/constants/fiatLogos.ts

```diff
@@ -1,13 +1,13 @@
-import type { FiatTypeEnumWithoutCryptos } from './fiats'
+import type { FiatCurrencyItem } from '@/components/Modals/FiatRamps/config'
```

Comment:
💜

Reply:
Noice new naming, though should prolly live somewhere shared, not fiat ramps modal?

Reply:
Did move to lib in [622b3b2](https://github.com/shapeshift/web/pull/10592/commits/622b3b2a98efda4bd32ce27229e2d08fcceff629)